ci/deps/azure-macos-35.yaml

```diff
@@ -22,7 +22,7 @@ dependencies:
   - xlsxwriter
   - xlwt
   # universal
-  - pytest
+  - pytest>=4.0.2
```

Comment:
i would add it to all files just to indicate we have a min version explicitly

Reply:
OK, if this succeeds, i'll do that.